calculator-app/src/main/java/camp/nextstep/edu/calculator/CalculatorActivity.kt

```diff
@@ -2,26 +2,35 @@ package camp.nextstep.edu.calculator
 
 import android.os.Bundle
 import android.widget.Toast
-import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModelProvider
+import camp.nextstep.edu.calculator.data.db.RecordDatabase
+import camp.nextstep.edu.calculator.data.repository.RecordRepositoryImpl
 import camp.nextstep.edu.calculator.databinding.ActivityCalculatorBinding
+import camp.nextstep.edu.calculator.domain.RecordRepository
 
 class CalculatorActivity : AppCompatActivity() {
     private lateinit var binding: ActivityCalculatorBinding
 
-    private val viewModel: CalculatorViewModel by viewModels()
+    private lateinit var viewModel: CalculatorViewModel
```

Comment:
```suggestion
    private val viewModel: CalculatorViewModel by viewModels {
        CalculatorViewModelProviderFactory()
    }
```

와 같은 형태로 변경할 수 있습니다..!

Reply:
최공최공!